lib/t/matches.ex

```diff
@@ -42,21 +43,32 @@ defmodule T.Matches do
     Phoenix.PubSub.broadcast_from(@pubsub, self(), pubsub_user_topic(user_id), message)
   end
 
-  # - Likes
+  defmacrop distance_km(location1, location2) do
+    quote do
+      fragment(
+        "round(? / 1000)::int",
+        st_distance_in_meters(unquote(location1), unquote(location2))
+      )
+    end
+  end
 
-  @spec like_user(Ecto.UUID.t(), Ecto.UUID.t()) ::
+  defp default_location, do: %Geo.Point{coordinates: {0, 0}, srid: 4326}
```

Comment:
?

Reply:
@akhrail1996 tests shouldn't affect the code they tests, please find another way